src/Microsoft.AspNetCore.Mvc.RazorPages/RazorPagesOptions.cs

```diff
@@ -134,6 +137,45 @@ public bool AllowMappingHeadRequestsToGetHandler
             set => _allowMappingHeadRequestsToGetHandler.Value = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value that determines if OPTIONS requests without a corresponding handler are handled by default
```

Comment:
suggestion `HTTP OPTIONS requests` or `OPTIONS HTTP requests` or `HTTP requests with the OPTIONS method`.